acme/src/acme/client.py

```diff
@@ -356,12 +356,7 @@ def renewal_time(self, cert_pem: bytes
             return None, now + default_retry_after
 
         ari_url = renewal_info_base_url + '/' + _renewal_info_path_component(cert)
-        try:
-            resp = self.net.get(ari_url, content_type='application/json')
-        except (requests.exceptions.RequestException, messages.Error) as error:
-            logger.info("failed to fetch renewal_info URL (%s): %s", ari_url, error)
-            return None, now + default_retry_after
-
+        resp = self.net.get(ari_url, content_type='application/json')
```

Comment:
blah i realize this is kinda counter to this whole approach, and i might be wrong, but don't we still wanna return the exponential backoff time if this raises a ["temporary"](https://www.ietf.org/archive/id/draft-ietf-acme-ari-08.html#section-4.3.3) exception?

if so, i wonder if we ought to catch the `RequestException` here, inspect what kinda error it was, and if it was temporary raise an exception w/ the backoff value attached?

Reply:
i realize we're not doing anything w/ the retry after value yet, but embedding that value now would let us do proper backoffs down the road

Reply:
FWIW when I wrote the "temporary" vs "long term" error distinction I was imagining that temporary errors would be retried during a single invocation of cron-based tools, for instance using automatic retries in the Requests library (or whatever style of retry is appropriate).

https://requests.readthedocs.io/en/latest/user/advanced/#example-automatic-retries